test/System/IntegrationTests/DynamoApplicationTests.cs

```diff
@@ -65,18 +65,6 @@ public void DynamoMakeModelWithHostName()
             Assert.AreEqual(model.HostName, "DynamoFormIt");
         }
 
-        [Test]
-        public void IfASMPathInvalidExceptionNotThrown()
```

Comment:
whats with this test?

Reply:
This tested the obsolete API, which I removed in this PR.

Reply:
Can't we just use the new API ? and keep the test